linux/ibus-keyman/icons/ibus-keyman.svg

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
We should find a better default icon as a cog is quite generic, perhaps Keyman for windows has one. Same goes for the default.png which currently shows `KMFL` as it's icon.

Reply:
> 
> 
> We should find a better default icon as a cog is quite generic, perhaps Keyman for windows has one. Same goes for the default.png which currently shows `KMFL` as it's icon.

removed, the svg isn't used anywhere.
updated the default.png from Logo320x234.png in team drive